src/content/docs/en/guides/astro-db.mdx

```diff
@@ -156,6 +163,180 @@ export default async function() {
+
+You can either use the CLI or the web interface to create and retrieve the values you need.
+The steps that are shown below will walk you through the process of installing the Turso CLI, logging in (or signing up), creating a new database, getting the two values we need and pushing our schema to the remote database.
+
+<Steps>
+
+1. Install the [Turso CLI](https://docs.turso.tech/cli/installation).
+
+2. [Login or signup](https://docs.turso.tech/cli/authentication) to Turso.
+
+3. Next, create a new database. In this example the database name is `andromeda`, but you can change it to any other name.
+
+   ```sh "andromeda"
+   turso db create andromeda
+   ```
+
+4. You can see the information about the newly created DB with the following command:
+
+   ```sh "andromeda"
+   turso db show andromeda
+   ```
+
+   Use the value from the row `URL:` as the environment variable `ASTRO_DB_REMOTE_URL`.
+   
+   :::tip
+   To read more about .env files and environment variables, check out the [Using environment variables](/en/guides/environment-variables/) guide.
+   :::
+
+   ```env title=".env" "libsql://andromeda-houston.turso.io"
+   ASTRO_DB_REMOTE_URL=libsql://andromeda-houston.turso.io
+   ```
+
+5. To authenticate requests to the database, create a new token for it:
+
+   ```sh "andromeda"
+   turso db tokens create andromeda
+   ```
+
+   Use the output of the command as the environment variable `ASTRO_DB_APP_TOKEN`
+
+   ```env title=".env" add={2} "eyJhbGciOiJF...3ahJpTkKDw"
+   ASTRO_DB_REMOTE_URL=libsql://andromeda-houston.turso.io
+   ASTRO_DB_APP_TOKEN=eyJhbGciOiJF...3ahJpTkKDw
+   ```
+
+6. Push your DB schema and metadata to the new Turso database.
+
+   ```sh
+   astro db push --remote
+   ```
+
+7. Congratulations, now you have a database connected! Give yourself a break. 👾
+
+   ```sh
+   turso relax
+   ```
+
+</Steps>
+
+To explore more features of Turso, check out the [Turso docs](https://docs.turso.tech).
+
+This was a quick guide on how to get started with Turso, read on to learn more on how to use Astro DB.
+
+### Connecting to remote databases
+
+Astro DB allows you to connect to both local and remote databases. By default, Astro uses a local database file for `dev` and `build` commands, recreating tables and inserting development seed data each time.
+
+To connect to a hosted remote database, use the `--remote` flag. This flag enables both readable and writable access to your remote database, allowing you to [accept and persist user data](#insert) in production environments.
+
+:::note
+While remote connections are generally possible with any deployment platform using static or server rendering mode, there are currently some limitations. Non-Node runtimes like Cloudflare and Deno don't currently support DB on server-rendered routes when using libSQL. Support for these platforms is planned for future implementation and the progress can be tracked by viewing [this PR](https://github.com/withastro/astro/pull/12163).
```

Comment:
Although I am in favor of adding a temporary aside until the implementation is finished (to reduce pressure on Fryuni 😄 ), I don't know if we should put a link to the PR. Perhaps we could cut the sentence after "for future implementation".

Reply:
something like this?

```suggestion
While remote connections are generally possible with any deployment platform using static or server rendering mode, there are currently some limitations. Non-Node runtimes like Cloudflare and Deno don't currently support DB on server-rendered routes when using libSQL. Support for these platforms is planned for future implementation.
```